src/RestAPI/dlwsrestapi.py

```diff
@@ -309,6 +309,26 @@ def get(self):
         return generate_response(ret)
 
 
+@api.resource("/ScaleInferenceJob")
```

Comment:
Maybe use `/ScaleJob`, do not be so specific about which job can be scaled in API